data/remnant/remnant ships.txt

```diff
@@ -892,37 +892,43 @@ ship "Merganser"
 		category "Interceptor"
 		licenses
 			Remnant
-		"cost" 6830000
-		"shields" 3000
-		"hull" 5000
+		"cost" 4830000
+		"shields" 3500
+		"hull" 3000
 		"required crew" 1
 		"bunks" 3
 		"cargo space" 11
 		"fuel capacity" 600
-		"ramscoop" 1.75
-		"mass" 30
-		"drag" 1.5
+		"ramscoop" 1.25
```

Comment:
```suggestion
		"ramscoop" 1.75
```
Keeping in line with warp core and Quantumshark, I think the ramscoop changes should be reverted and included in a separate PR that deals with all of the Remnant integrated ramscoops, however, I am fine with this PR being merged without those changes, as long as this is kept in mind when the other ships are looked at.